x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -347,6 +347,14 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         describe('Processes Tab', () => {
           before(async () => {
             await pageObjects.assetDetails.clickProcessesTab();
+            await pageObjects.header.waitUntilLoadingHasFinished();
+          });
+
+          it('should render processes tab and with Total Value summary', async () => {
+            const processesTotalValue =
+              await pageObjects.assetDetails.getProcessesTabContentTotalValue();
+            const processValue = await processesTotalValue.getVisibleText();
+            expect(processValue).to.eql('313');
```

Comment:
We already have this test in the [node_details suite](https://github.com/elastic/kibana/blob/main/x-pack/test/functional/apps/infra/node_details.ts#L238). Do you think it's worth it repeating it for the flyout?

Reply:
Maybe it's not needed here anymore but we had it before so I guess I should open this PR only for older versions (8.9 where it failed) so I will check that. As we will have different content on the flyout and the full page it will make sense to test the different sections and to have the main tests in one place (the processes tab is the same for now so it's fine)